docs/Discord_MERLIN_for_Wind.ipynb

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
<p>What is this file? Should it be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Sorry for this! Is that the only new file you see? I have it but I do not remember `add`ing it to my branch. Going to delete all unnecessary files on my end.

Reply:
I think you'll have to delete it and then perform a `git rm docs/Discord_MERLIN_for_Wind.ipynb` and push that commit. I only noticed it because it said 2 files were changed:
<img width="1440" alt="Screen Shot 2022-02-01 at 9 23 17 AM" src="https://user-images.githubusercontent.com/7473521/151986465-61e54b4c-46d0-414d-882e-65b929fe1880.png">

Reply:
Yeah. I know. I usually avoid `git add .` to not add all stuff. I usually do `add notebook`. I will remove and push it in the next set of commits.